migrate/20240218_add_concession.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  change do
+    create_table(:concession) do
+      column :id, :uuid, primary_key: true, default: Sequel.lit("gen_random_uuid()")
+      column :project_id, :project, type: :uuid, null: false
+      column :resource_type, :text, collate: '"C"'
+      column :credit, :numeric, null: false, default: 0
+      constraint(:min_credit) { credit >= 0 }
+      column :discount, :Integer, null: false, default: 0
+      constraint(:max_discount) { discount <= 100 }
+    end
```

Comment:
If you add one more column; recurring_till, we can issue more complex concessions, such as;
- 1$ off on Github Runner each month till +infinity
- 50% off on Github Runners for one year

(both are currently happening cases)